fs/dafsclient/rmtfile.cpp

```diff
@@ -1570,7 +1570,7 @@ IFileIO * CRemoteFile::open(IFOmode mode,IFEflags extraFlags)
 void CRemoteFile::copyTo(IFile *dest, size32_t buffersize, ICopyFileProgress *progress, bool usetmp, CFflags copyFlags)
 {
     CRemoteFile *dstfile = QUERYINTERFACE(dest,CRemoteFile);
-    if (dstfile&&!dstfile->queryEp().isLocal()) {
+    if (dstfile&&(!dstfile->queryEp().isLocal() || (dstfile->queryEp().port!=DAFILESRV_PORT && dstfile->queryEp().port!=SECURE_DAFILESRV_PORT))) {
```

Comment:
Just a comment, I don't see any relation, other than a convention made, between port number and host locality.
But its good that this test matches the other pre-existing logic.

Reply:
Yes, remote files access is only used for non local IP's OR if the URL explicitly references a non-standard port.

This change is now consistent with how createIFile has always behaved.